EIPS/eip-2020.md

```diff
@@ -0,0 +1,232 @@
+---
+eip: 2020
+title: EM Token
+author: Julio Faura <julio@adhara.io>, Fernando Paris <fer@io.builders>, Daniel Lehrner <daniel@io.builders>
+discussions-to: https://github.com/IoBuilders/EIPs/issues/6
```

Comment:
I suggest to create a discussion URL on this repo, just like you did for the other EIPs.